pkg/tpm/tcpa_log.go

```diff
@@ -0,0 +1,730 @@
+package tpm
+
+import (
+	"bytes"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"unicode/utf16"
+)
+
+/*
```

Comment:
This would be good in a package comment?

Package comments are chosen to be in *one* file in the package, and go like this:

```go
// License statement

// Package tpm adds UEFI-specific TPM measurement bla bla bla
//
// [1] ...
// [2] ...
package tpm
```

must be directly above the `package tpm` line.